ofl/signika/DESCRIPTION.en_us.html

```diff
@@ -1,12 +1,18 @@
 <p>
-Signika is a sans-serif with a gentle character, developed for wayfinding, signage, and other media where clarity of information is required. 
-It has a low contrast and tall x-height to improve readability of texts in small sizes as well as in large distances from the reader. 
-Being a typical signage typeface it is inspired by typefaces such as Ronnia, Meta, and Tahoma.
+    <a href="https://fonts.google.com/specimen/Signika">Signika</a> is a sans-serif with a gentle character, developed
```

Comment:
We don't need this link since the user will already be on this page when viewing this description.

Reply:
Ammended that, thanks for pointing it out.